python/instrumentation/openinference-instrumentation-llama-index/src/openinference/instrumentation/llama_index/__init__.py

```diff
@@ -30,40 +27,28 @@ def instrumentation_dependencies(self) -> Collection[str]:
     def _instrument(self, **kwargs: Any) -> None:
         if not (tracer_provider := kwargs.get("tracer_provider")):
             tracer_provider = trace_api.get_tracer_provider()
-        self._use_experimental_instrumentation = kwargs.get("use_experimental_instrumentation")
-        if self._use_experimental_instrumentation:
-            if not _legacy_llama_index():
-                print(
-                    "`use_experimental_instrumentation` feature flag is set. Spans "
-                    "will be generated using the new instrumentation system "
-                    "For more information about the new instrumentation system, visit "
-                    "https://docs.llamaindex.ai/en/stable/module_guides/observability/instrumentation/"  # noqa E501
-                )
+        self._use_legacy_callback_handler = kwargs.get("use_legacy_callback_handler")
+        if self._use_legacy_callback_handler:
+            import llama_index.core
+
+            if hasattr(llama_index.core, "global_handler"):
+                print("Using legacy callback handler.")
             else:
-                print(
-                    f"`use_experimental_instrumentation` feature flag is set. But "
-                    f"the version of `llama-index-core` is not "
-                    f">={'.'.join(map(str, _ELIGIBLE_VERSION_FOR_NEW_INSTRUMENTATION))}, "
-                    f"so the flag is ignored."
-                )
+                print("Legacy callback handler is not available.")
```

Comment:
```suggestion
                print("Legacy callback handler is not available. Using new callback handler.")
```

Suggestion: A slightly more verbose message would be helpful here.

Reply:
ok